### PR TITLE
fix(rpc): get_proof test fail because block numbers are not high enough

### PR DESCRIPTION
**Pull request by yihuang**
## Description

Solution:
- wait for app block height rather than tendermint block height

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: tests/integration_tests/test_types.py [hunk 1]
```diff
@@ -317,6 +319,8 @@ def test_estimate_gas(ethermint, geth):
 def make_same_rpc_calls(rpc1, rpc2, method, params):
     res1 = rpc1.make_request(method, params)
     res2 = rpc2.make_request(method, params)
+    print("res1", res1)
+    print("res2", res2)
```
Comment (yihuang) [inline tests/integration_tests/test_types.py:323]:
```suggestion
```

**Review by danburck — APPROVED** [review]
@yihuang LGTM, lets merge this. And I'll update my PR: https://github.com/evmos/ethermint/pull/1233

**Review by ramacarlucho — APPROVED** [review]

**Comment by danburck** [comment]
@yihuang Integration tests are still failing

**Comment by yihuang** [comment]
> @yihuang Integration tests are still failing

fixed, this time it's a real bug.

**Review by fedekunze — APPROVED** [review]

**Pull request merged**